src/test_l1_dsHdmiIn.c

```diff
@@ -2016,7 +2029,13 @@ void test_l1_dsHdmiIn_positive_dsGetSupportedGameFeaturesList(void) {
     UT_ASSERT_EQUAL(dsGetSupportedGameFeaturesList(&supported_features_1), dsERR_NONE);
 
     // Step 3: Call dsGetSupportedGameFeaturesList() with valid inputs (dsSupportedGameFeatureList_t*)
-    UT_ASSERT_EQUAL(dsGetSupportedGameFeaturesList(&supported_features_2), dsERR_NONE);
+    ut_kvp_getStringField(instance, "dsHdmiIn/gameFeatures/feature", featureString, sizeof(featureString));
+
+    strncpy(supported_features_2.gameFeatureList, featureString, sizeof(supported_features_2.gameFeatureList) - 1);
+    supported_features_2.gameFeatureList[sizeof(supported_features_2.gameFeatureList) - 1] = '\0';
```

Comment:
Used a macro everywhere for this `sizeof(supported_features_2.gameFeatureList)` and take -1 off it in this case..

Reply:
Done